library/create_kafka.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/python
+
+
+from __future__ import (absolute_import, division, print_function)
+from sys import api_version
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: create_kafka
+
+short_description: Create Red Hat OpenShift Streams for Apache Kafka Instance
+
+# If this is part of a collection, you need to use semantic versioning,
+# i.e. the version is of the form "2.5.0" and not "2.4".
+version_added: "0.1.0"
+
+description: Create Red Hat OpenShift Streams for Apache Kafka Instance
+
+options:
+    name:
+        description: Name of the Kafka instance
+        required: true
+        type: str
+    cloud_provider:
+        description: Cloud provider for the Kafka instance
+        required: false
+        type: str
+        default: aws
+    region:
+        description: Region of the Kafka instance
+        required: false
+        type: str
+        default: eu-west-1
+    reauthentication_enabled:
+        description: Reauthentication enabled for the Kafka instance
+        required: false
+        type: bool
+        default: true
+    plan:
+        description: Plan for the Kafka instance
+        required: false
+        type: str
+        default: ""
+    billing_cloud_account_id:
+        description: Billing cloud account id for the Kafka instance
+        required: false
+        type: str
+        default: ""
+    marketplace:
+        description: Marketplace for the Kafka instance
+        required: false
+        type: str
+        default: ""
+    billing_model:
+        description: Billing model for the Kafka instance
+        required: false
+        type: str
+        default: marketplace
+    instance_type:
+        description: Instance type for the Kafka instance
+        required: false
+        type: str
+        default: developer
+ 
+# Specify this value according to your collection
+# in format of namespace.collection.doc_fragment_name
+extends_documentation_fragment:
+    - my_namespace.my_collection.my_doc_fragment_name
+
+author:
+    - Red Hat Developer
+'''
+
+EXAMPLES = r'''
+# Pass in a message
+  - name: Create kafka
+    create_kafka:
+      name: "struttin"
+    register:
+      kafka_req_resp 
+
+ 
+'''
+
+RETURN = r'''
+# These are examples of possible return values, and in general should use other names for return values.
```

Comment:
We have option to limit return values? Returning API values that we do not control is risky. 
I think we can point user to autogenerated sdk documentation for most up to date values or 
map values in ansible. I think I will pick to not document values as those will outdate faster than sandwitch

Reply:
yea, we can return what ever values / subset of values we want. I stuck them all in here for verbosity but yea I can just as easy remove them. 

you're idea of referencing the SDK docs here is a much better idea that what I'm doing here